src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -143,4 +209,26 @@ def parse_args(argv=None):
     add_dataset_args(dataset_group)
 
     args = parser.parse_args(argv)
+
+    # Concurrency and request rate are mutually exclusive
+    # TODO: Review if there is a cleaner way to do this with argparse
+    if args.concurrency is not None and args.request_rate is not None:
+        parser.error(
+            "Arguments --concurrency and --request_rate are mutually exclusive."
```

Comment:
Might want simpler terms. Just say you can't specify both options?

Reply:
I am actually not sure if this can be a mutual exclusive group in this argument group or if i need to pull it out to its own thing.
@rmccorm4 do you know?

Reply:
I didn't quite follow 😅. I was just suggesting different error message

Reply:
I think you can do something like this:
```
# Set required=True on the group if you require one of them *must* be specified
group = parser.add_mutually_exclusive_group(required=True)
group.add_argument('--concurrency', ...)
group.add_argument('--request-rate', ...)

# Should automatically error out if both are provided.
# If required=True, will also error out if neither are provided.
```

Reply:
Added a mutually exclusive subgroup to the profile group that include concurrency and request rate.